static/app/views/performance/transactionSummary/transactionSpans/content.tsx

```diff
@@ -233,8 +245,11 @@ function SpansContentV2(props: Props) {
         <StyledSearchBar
           organization={organization}
           projectIds={eventView.project}
-          query={''}
+          query={query.query}
           fields={eventView.fields}
+          placeholder={t('Search for span attributes')}
+          supportedTags={supportedTags}
+          dataset={DiscoverDatasets.SPANS_INDEXED}
```

Comment:
maybe add a comment here to explain why we're using SPANS_INDEXED even though the search is querying SPANS_METRICS

Reply:
Good point, I guess the reason isn't immediately obvious